instrument-starters/opentracing-spring-cloud-reactor-starter/src/main/java/io/opentracing/contrib/spring/cloud/reactor/ReactorTracingAutoConfiguration.java

```diff
@@ -37,27 +45,74 @@
  * @author Csaba Kos
  */
 @Configuration
-@AutoConfigureAfter(TracerAutoConfiguration.class)
```

Comment:
Should we have an `@AutoConfigureBefore(...)` or `@Order(...)` annotation here to make sure that this gets loaded "early enough"?

Reply:
I am not sure that's necessary. As you have mentioned before, we can not prevent everything from happening. Someone could use Reactor in the configuration to easily break it (As I can easily break Sleuth implementation as well by using Reactor early enough). Besides, I could even put back this line `@AutoConfigureAfter(TracerAutoConfiguration.class)`, and the implementation should work without any problems.

Reply:
Is removing this necessary? We use this in every instrumentation starter

Reply:
@pavolloffay I can't see why this is necessary... But you think it's better to have this consistency, I could put it back.

Reply:
> The post processor will run anyway before the Tracer bean is available.

Should be also `@ConditionalOnBean(Tracer.class)`  removed then?